packages/browser-crawler/src/internals/browser-crawler.ts

```diff
@@ -171,6 +176,29 @@ export interface BrowserCrawlerOptions<
      */
     handlePageFunction?: BrowserCrawlerHandleRequest<Context>;
 
+    /**
+     * A function to handle requests that failed less than `option.maxRequestRetries` times
```

Comment:
i'd just say its a function for every retry, and we should say explicitly that it allows modifying the request object before it gets retried